django_etuovi/items.py

```diff
@@ -117,8 +116,8 @@ class Item(BaseClass):
     rc_parkingspace_count: int
     roomcount: int
     scontact: List[Scontact]
-    showingdate: date
-    showing_date2: date
+    showingdate: str
+    showing_date2: str
```

Comment:
Wouldn't it make more sense to define the type as `datetime` and then handle the formatting during serialization instead of expecting correctly formatted string timestamps up front?

Reply:
Yep, that sounds like a better idea. I'll change.

Reply:
This is fixed now.